Snakefile

```diff
@@ -62,6 +79,11 @@ wildcard_constraints:
     ll="(v|c)([0-9\.]+|opt|all)|all",
     opts="[-+a-zA-Z0-9\.]*",
     unc="[-+a-zA-Z0-9\.]*",
+    sopts="[-+a-zA-Z0-9\.\s]*",
+    discountrate="[-+a-zA-Z0-9\.\s]*",
+    demand="[-+a-zA-Z0-9\.\s]*",
+    h2export="[0-9]+m?|all",
+    planning_horizons="20[2-9][0-9]|2100",
```

Comment:
Ok for now, but worth adding an issue to simplify this

Reply:
thanks for coming back to this, I raised an issue to tackle that later